src/tools/ApiChecker/CheckerOptions.cs

```diff
@@ -0,0 +1,10 @@
+using CommandLine;
+
+namespace ApiChecker
+{
+    [Verb("check-api", isDefault: true)]
+    internal class CheckerOptions
+    {
+        // this is empty for now
```

Comment:
```suggestion
```

Reply:
Let's open an issue and resolve separately. I'm conscious of keeping the PR as minimal as possible :)

Reply:
https://github.com/microsoft/playwright-sharp/issues/1010